### PR TITLE
Fix part of #1874: Fix AdministratorControlsActivityTest for robolectric

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #1874 
Fixed `AdministratorControlsActivityTest`

Screenshot 2020-11-05 at 10 51 53
Screenshot 2020-11-05 at 10 50 06

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 1]
```diff
@@ -46,64 +44,101 @@ import org.junit.Ignore
...
 
   private fun setUpTestApplicationComponent() {
-    DaggerAdministratorControlsActivityTest_TestApplicationComponent.builder()
-      .setApplication(ApplicationProvider.getApplicationContext())
-      .build()
-      .inject(this)
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  // TODO(): Implement `isFromNavigationDrawer` in AdministratorControlsActivity. Reference - HelpActivity
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: @rt4914 need to discuss these 2 ignored test cases.

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Removed both e2e tests, as for one change config is already there and other we cannot test closing activity as there is no other activity in the stack while testing, so the app will be killed.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 1 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Resolving this on your behalf as we had conversation over call about this.

**Pull request merged**